WORKSPACE

```diff
@@ -115,6 +97,28 @@ github_deps()
 load("@vaticle_bazel_distribution//pip:deps.bzl", pip_deps = "deps")
 pip_deps()
 
+######################################
+# Load @vaticle_dependencies//distribution/docker #
+######################################
+
```

Comment:
I think this technical debt is severe enough to warrant an issue, and a code comment.
```suggestion

# must be loaded after `vaticle_bazel_distribution` to ensure
# `rules_pkg` is correctly patched (bazel-distribution #issue_no)

```
And raise an issue in `bazel-distribution`, something like "The `rules_pkg` patch to allow long filenames is fragile".

The issue description should say that the patch needs to be applied before repos such as `rules_docker` are loaded in (that themselves depend on `rules_pkg`), and we should reference https://github.com/vaticle/bazel-distribution/issues/251 in the issue.

Reply:
Created the issue: https://github.com/vaticle/bazel-distribution/issues/355.